test/evp_kdf_test.c

```diff
@@ -239,40 +240,75 @@ static int test_kdf_hkdf(void)
     return ret;
 }
 
-static int do_kdf_hkdf_gettables(int expand_only, int has_digest)
+static int do_kdf_hkdf_gettables(int extract_only, int has_digest)
```

Comment:
Despite this previously being `expand_only`, when construct_hkdf_params() gets a NULL inform parameter it created an EXTRACT_ONLY KDF.